Makefile

```diff
@@ -36,7 +36,7 @@ BUILD_OS := $(shell uname -s)
 
 # Set the version information for the Kubernetes servers, and build localkube statically
 K8S_VERSION_LDFLAGS := $(shell $(PYTHON) hack/get_k8s_version.py 2>&1)
-MINIKUBE_LDFLAGS := -X k8s.io/minikube/pkg/version.version=$(VERSION)
+MINIKUBE_LDFLAGS := $(K8S_VERSION_LDFLAGS) -X k8s.io/minikube/pkg/version.version=$(VERSION)
```

Comment:
I have no idea how that could cause a panic - seems a bit dangerous if that is the case - but you're right about the duplicate linker options which could be the cause. Updated now.